index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
+      <ol>
+        <li>Exactly one "opus" codec, with <code>numChannels</code> set to 2,
```

Comment:
Well, if I understand correctly, that's what you _need_ to do with SDP if you want mono audio with Opus. If ORTC can avoid that oddity, that's great, it's just one of the differences from SDP I need to remember.